python/usb_console/radio_session.py

```diff
@@ -45,169 +51,53 @@ def __init__(self, device_name, datastore, logger, radio_keys_config):
-                                            
-
-                            #check if there is an email attachment
-                            if part.get_filename() is not None:
-                                #get data from email attachment
-                                attachmentContents=part.get_payload(decode=True).decode('utf8')
-                                data=self.process_downlink_packet(attachmentContents)
-                                
-                                #iterate through each statefield key of the dictionary
-                                entry={}
-                                for key in self.statefields:
-                                    #if the value for a statefield exists in the json object recieved, update the dictionary and add entry to datastore
-                                    if data.get(key)is not None:
-                                        self.statefields[key]=data[key]
-                                        entry['field'] = key
-                                        entry['val'] = data[key]
-                                        entry['time'] = datetime.datetime.now()
-                                        self.datastore.consume_queue_item(entry)
-
     def read_state(self, field, timeout=None):
         '''
-        Read state.
-        
-        Read the latest value of the state field received via downlink with the name "field".
+        Read state by posting a request for data to the Flask server
         '''
-        return self.statefields[field]
+        headers = {
+            'Accept': 'text/html',
+        }
+
+        response = requests.get('http://'+self.flask_server+':'+self.flask_port+'/search-statefields?imei='+str(self.imei)+'&field='+str(field), headers=headers)
+        return response.text
 
     def write_multiple_states(self, fields, vals, timeout=None):
         '''
         Uplink multiple state variables. Return success of write.
+        Reads from the most recent Iridium Report whether or
+        not RadioSession is able to send uplinks
         '''
         assert len(fields) == len(vals)
 
-        #create dictionary object with new fields and vals
-        updated_fields={}
-        for i in range(len(fields)):
-            updated_fields[fields[i]]=vals[i]
-
-        #connect to PAN email account
-        yag = yagmail.SMTP(self.username, self.password)
-        #create a JSON file with the updated statefields and send it to the iridium email
-        with open('uplink.json', 'w') as json_uplink:
-            json.dump(updated_fields, json_uplink)
-        if self.send_uplinks==True:
-            yag.send('sbdservice@sbd.iridium.com', 'Data', 'uplink.json')
+        headers = {
+            'Accept': 'text/html',
+        }
+
+        response = requests.get('http://'+self.flask_server+':'+self.flask_port+'/search-iridium?imei='+str(self.imei)+'&field=send-uplinks', headers=headers)
+        print(response.text)
```

Comment:
Can you delete this print statement?

Reply:
Actually, this information is useful for debugging. Can you instead add it to `self.logger`